### PR TITLE
Improve Docker Stack

### PR DESCRIPTION
**Pull request by cebe**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️
| Breaks BC?    | probably yes but docker stack was not really working before
| Fixed issues  | -

### CODE REVIEW

**Comment by xepozz** [comment]
1. Isn’t better to use php 8.2?
2. Have you tested package linking?

File: docker/home/.bashrc [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    #alias grep='grep --color=auto'
+    #alias fgrep='fgrep --color=auto'
+    #alias egrep='egrep --color=auto'
+fi
+
+# colored GCC warnings and errors
+#export GCC_COLORS='error=01;31:warning=01;35:note=01;36:caret=01;32:locus=01:quote=01'
+
+# some more ls aliases
+#alias ll='ls -l'
```
Comment (samdark) [inline docker/home/.bashrc:91]: I'd enable these.

**Comment by cebe** [comment]
> 1. Isn’t better to use php 8.2?

if everything works fine on 8.2, we can change it to that, might even be useful to be able to switch between php versions.

> 2. Have you tested package linking?

yes, works

image

File: docker/home/.bashrc (hunk 1 above)
Comment (cebe) [reply docker/home/.bashrc]: done.

**Comment by cebe** [comment]
Updated PHP version to 8.2 and documented how to change PHP version.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thanks!

**Pull request merged**